hypervisor/arch/x86/guest/vlapic.c

```diff
@@ -883,6 +885,8 @@ vlapic_process_eoi(struct acrn_vlapic *vlapic)
 			/* hook to vIOAPIC */
 			vioapic_process_eoi(vlapic->vm, vector);
 		}
+
```

Comment:
Could we just call vcpu_make_request(vlapic->vcpu, ACRN_REQUEST_EVENT); here ?

Reply:
new PR according to fei's comment